pyproject.toml

```diff
@@ -18,4 +18,9 @@ requires-python = ">=3.10"
 readme = "README.md"
 license = {text = "MIT"}
 
+[project.scripts]
+parse-contributors = "pyosmeta.cli.parse_contributors:main"
```

Comment:
Nice :sunny:

Reply:
oh wow - is this an entry point? i've read a bit about this but am learning here and REALLY appreciate that!! :)